<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1469</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lOptitrackX">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>xPosOptitrack</string>
    </property>
   </widget>
   <widget class="QLabel" name="lOptitrackY">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>yPosOptitrack</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>481</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>513</x>
      <y>10</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Chancery L</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>treXton evaluation</string>
    </property>
   </widget>
   <widget class="QLabel" name="xPos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x-Position:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yPos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y-Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="lOptitrack">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Optitrack</string>
    </property>
   </widget>
   <widget class="QLabel" name="ltreXton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>treXton</string>
    </property>
   </widget>
   <widget class="QLabel" name="lTrexton_x">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>xPosTreXton</string>
    </property>
   </widget>
   <widget class="QLabel" name="lTrexton_y">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>yPosTreXton</string>
    </property>
   </widget>
   <widget class="QLabel" name="lUncertainty">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Uncertainty</string>
    </property>
   </widget>
   <widget class="QLabel" name="confVal_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>confVal</string>
    </property>
   </widget>
   <widget class="QLabel" name="confVal_y">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>confVal</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>353</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCustomPlot" name="confBar" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>531</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="histogram" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>641</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="distPlot" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>460</y>
      <width>541</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lDistance">
    <property name="geometry">
     <rect>
      <x>1086</x>
      <y>333</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Euclidean distance</string>
    </property>
   </widget>
   <widget class="QLabel" name="dist_Val">
    <property name="geometry">
     <rect>
      <x>1156</x>
      <y>380</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>dist_Val</string>
    </property>
   </widget>
   <widget class="QLabel" name="dist_Mean">
    <property name="geometry">
     <rect>
      <x>1156</x>
      <y>400</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>dist_Mean</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCurrent">
    <property name="geometry">
     <rect>
      <x>1056</x>
      <y>380</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>current:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lMean">
    <property name="geometry">
     <rect>
      <x>1056</x>
      <y>400</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mean:</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>1036</x>
      <y>353</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lCorr">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>335</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Correlation between distance and confidence</string>
    </property>
   </widget>
   <widget class="QLabel" name="corrVal">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>381</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>dist_Val</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>353</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>22</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Chancery L</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
color: rgb(24, 136, 0);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1469</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
